UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Services/FssService.cs

```diff
@@ -56,6 +56,11 @@ public async Task<FssBatchStatus> CheckIfBatchCommitted(string batchId, RequestT
                 batchStatusPollingCutoffTime = double.Parse(_fssApiConfiguration.Value.BatchStatusPollingCutoffTime);
                 batchStatusPollingDelayTime = int.Parse(_fssApiConfiguration.Value.BatchStatusPollingDelayTime);
             }
+            else if (requestType.Equals(RequestType.BESS))
+            {
+                batchStatusPollingCutoffTime = double.Parse(_fssApiConfiguration.Value.BatchStatusPollingCutoffTimeForBES);
+                batchStatusPollingDelayTime = int.Parse(_fssApiConfiguration.Value.BatchStatusPollingDelayTimeForBES);
+            }
```

Comment:
this is not in the stryker report or at least i don't see the stryker report for the latest execution

Reply:
added required changes.